examples/packer-azure-windows/windows.json

```diff
@@ -0,0 +1,71 @@
+{
+  "variables": {
+    "client_id": "{{env `ARM_CLIENT_ID`}}",
+    "client_secret": "{{env `ARM_CLIENT_SECRET`}}",
+    "resource_group": "{{env `ARM_RESOURCE_GROUP`}}",
+    "subscription_id": "{{env `ARM_SUBSCRIPTION_ID`}}",
+    "tenant_id": "{{env `ARM_TENANT_ID`}}"
+  },
+  "builders": [
+    {
+      "type": "azure-arm",
+        
+      "client_id": "{{user `client_id`}}",
+      "client_secret": "{{user `client_secret`}}",
+      "subscription_id": "{{user `subscription_id`}}",
+      "tenant_id": "{{user `tenant_id`}}",
+
+      "managed_image_resource_group_name": "{{user `resource_group`}}",
+      "managed_image_name": "myPackerImage",
+
+      "os_type": "Windows",
+      "image_publisher": "MicrosoftWindowsServer",
+      "image_offer": "WindowsServer",
+      "image_sku": "2016-Datacenter",
+
+      "communicator": "winrm",
+      "winrm_use_ssl": true,
+      "winrm_insecure": true,
+      "winrm_timeout": "15m",
+      "winrm_username": "packer",  
+
+      "azure_tags": {
+        "dept": "engineering",
+        "task": "image deployment"
+      },
+
+      "build_resource_group_name": "{{user `resource_group`}}",
+      "vm_size": "Standard_D2_v3"
+    }
+  ],
+  "provisioners": [
+    {
+      "type": "powershell",
+      "elevated_user": "packer",
+      "elevated_password": "{{.WinRMPassword}}",
+      "inline": [
+        "# Turn off PowerShell execution policy restrictions",
+        "Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope LocalMachine",
+        "# Install Chocolatey",
+        "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))",
+        "# Install Docker Version Manager",
```

Comment:
@carolynvs thank you for the suggestion to try dvm!  Indeed, installation appears to go great on packer build and the azure vm is successfully created, but I'm currently greeted with a black screen when RDP-ing into the subsequent VM.  I'm sure this isn't due to dvm, but just wanted to update on progress.  Exciting!